src/App.jsx

```diff
@@ -14,70 +15,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedName, setselectedName] = useState('Jam');
```

Comment:
it shoud be upperCase update to `setSelectedName`

also think about naming...selectedName is not the best name, currently you understand what is it, but in more complex project you're gonna have a lot of diffenet elements and you have to understand what is `selectedName`